src/service/poc_mobile.proto

```diff
@@ -692,6 +700,24 @@ message invalid_data_transfer_ingest_report_v1 {
   uint64 timestamp = 3;
 }
 
+message pending_data_transfer_session_v1 {
+  // Public Key of the Hotspot
+  bytes pub_key = 1;
+  // Public Key of the Payer
+  bytes payer = 2;
+
+  uint64 upload_bytes = 3;
+  uint64 download_bytes = 4;
+  uint64 rewardable_bytes = 5;
+
+  // Timestamp in seconds since the epoch
+  // From the DataTransferEvent
+  uint64 event_timestamp = 6;
+  // Timestamp in milliseconds since the epoch
+  // From the file the transfer session was processed
+  uint64 received_timestamp = 7;
```

Comment:
One timestamp in seconds, but another in milliseconds can be confusing.
Is it possible to use one format?

Reply:
Agreed.
I was trying to respect the format of the message the timestamps originated from, but I'm not sure that adds anything beneficial. 

Moving to millis only.